docs/reference/api/docker_remote_api_v1.21.md

```diff
@@ -1881,10 +1881,14 @@ Docker containers report the following events:
 
     attach, commit, copy, create, destroy, die, exec_create, exec_start, export, kill, oom, pause, rename, resize, restart, start, stop, top, unpause
 
-and Docker images report:
+Docker images report:
 
     delete, import, pull, push, tag, untag
 
+Docker builds report:
+
+    build
+
```

Comment:
Not in docs review yet, but could you update https://github.com/docker/docker/blob/master/docs/reference/commandline/events.md and https://github.com/docker/docker/blob/master/man/docker-events.1.md as well?